store/sqlstore/user_store.go

```diff
@@ -707,65 +669,58 @@ type UserWithLastActivityAt struct {
 
 func (us SqlUserStore) GetRecentlyActiveUsersForTeam(teamId string, offset, limit int) store.StoreChannel {
 	return store.Do(func(result *store.StoreResult) {
-		var users []*UserWithLastActivityAt
+		query := usersQuery.
+			Select("s.LastActivityAt").
+			Join("TeamMembers tm ON (tm.UserId = u.Id AND tm.TeamId = :TeamId)").Bind("TeamId", teamId).
+			Join("Status s ON (s.UserId = u.Id)").
+			OrderBy("s.LastActivityAt DESC").
+			OrderBy("u.Username ASC").
```

Comment:
Adding an explicit ordering here in addition to `s.LastActivityAt` to simplify unit testing and remove non-determinism, as above.